index.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+require __DIR__ . '/bootstrap/simple-errors.php';
+
+displaySimpleError('ROOT', 403, '<span class="important">This is the root directory and it MUST NOT BE PUBLICALLY ACCESSIBLE.</span><br>
```

Comment:
Am I correct that the 403 is just for show? :(

Reply:
what do you mean ?

Reply:
Does it return HTTP 403 somehow?

Reply:
( https://www.php.net/manual/en/function.http-response-code.php )

Reply:
done.